http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -197,7 +198,7 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
         .getNotebookServiceAccount(userInfo, googleProject)
       serviceAccountInfo = ServiceAccountInfo(clusterServiceAccountOpt, notebookServiceAccountOpt)
 
-      clusterOpt <- dbRef.inTransactionIO { _.clusterQuery.getActiveClusterByNameMinimal(googleProject, clusterName) }
+      clusterOpt <- dbRef.dataAccess.clusterQuery.getActiveClusterByNameMinimal(googleProject, clusterName).transaction
```

Comment:
this will be preferred way of converting `DBIO` to `IO`...most other places are doing 
```
dbRef.inTransaction {dbRef.dataAccess.clusterQuery.getActiveClusterByNameMinimal(googleProject, clusterName)}
```
which is a bit more verbose, they are that way for now because I introduced the syntax after I made all the other changes. Manual change to all the other places is a bit labor intense, hence leaving them as is for now.